### PR TITLE
logs link added to pod detail

### PR DESCRIPTION
**Pull request by batikanu**
Fix for #1174
A link for logs is added to the pod details page

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/dashboard/1195)

### CODE REVIEW

**Comment by maciaszczykm** [comment]
@batikanu Please take a look on my comment here: https://github.com/kubernetes/dashboard/issues/1174#issuecomment-244886646

**Comment by batikanu** [comment]
What about this?
podscontainers

**Comment by bryk** [comment]
I'd just go with `Logs` or `View logs`. No reason to add brackets `[]` or Start Words With Capital Case :) 

@maciaszczykm @floreks ?

**Comment by maciaszczykm** [comment]
I like `View logs` option.

**Comment by batikanu** [comment]
Done. PTAL

File: src/app/frontend/poddetail/containerinfo.html [hunk 1]
```diff
@@ -56,5 +56,11 @@
         {{::$ctrl.i18n.MSG_CONTAINER_DETAILS_NO_ARGS}}
       </div>
     </kd-info-card-entry>
+    <kd-info-card-entry>
```
Comment (bryk) [inline src/app/frontend/poddetail/containerinfo.html]: AFAIK containerinfo.html is also used from replication controllers and others. You should do checking whether podName is specified.

**Comment by bryk** [comment]
One comment. PTAL

**Comment by batikanu** [comment]
PTAL

---

Review status: 0 of 8 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/app/frontend/poddetail/containerinfo.html, line 59 \[r1\]](https://reviewable.io:443/reviews/kubernetes/dashboard/1195#-KQzTmb_mxl7PenmSdrR-r1-59:-KR2rbgCEQQyVNAt5ecc:b-896fix) ([raw file](https://github.com/kubernetes/dashboard/blob/ef82d69eb76700b15fdcb39d810731e76decd926/src/app/frontend/poddetail/containerinfo.html#L59)):*
<details><summary><i>Previously, bryk (Piotr Bryk) wrote…</i></summary>
> AFAIK containerinfo.html is also used from replication controllers and others. You should do checking whether podName is specified. 
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/1195#-:-KR2rdQoqOt84SfOYhnM:bb71pro)*

**Comment by bryk** [comment]
:lgtm: Thanks! 

---

Reviewed 5 of 8 files at r1, 3 of 3 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/1195#-:-KR3oPVwa-1_Lq00QMzG:b-qyabt)*

**Pull request merged**